packages/playground/src/weblets/tf_deployment_list.vue

```diff
@@ -187,9 +187,15 @@
           />
           <IconActionBtn
             tooltip="Open Nextcloud"
+            color="white"
```

Comment:
Please remove the static color here, it should be dynamic

see in dark mode 

- ![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/57001890/8ef23057-95b5-45b5-b340-246cbe17f0c3)

light mode 
- ![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/57001890/16aeead5-cff7-42d1-84b6-eaa2ae62825f)